modules/rest-api/pages/rest-cluster-addnodes.adoc

```diff
@@ -95,7 +95,14 @@ See xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].
 [#see-also]
 == See Also
 
-For a conceptual overview of nodes, including options for adding nodes to clusters, see xref:learn:clusters-and-availability/nodes.adoc[Nodes].
-For information on rebalancing, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].
-For information on adding nodes and rebalancing by means of Couchbase Web Console and the CLI, see xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a Node and Rebalance].
-For information on correctly specifying the node-name on its certificate, see xref:learn:security/certificates.adoc#node-certificate-validation[Node-Certificate Validation].
+* For a conceptual overview of nodes, including options for adding nodes to clusters, see xref:learn:clusters-and-availability/nodes.adoc[Nodes].
+
+* For information on rebalancing, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].
```

Comment:
`* For information on rebalancing using the REST API, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].`